.github/workflows/update-azure-devops-templates-from-upstream.yaml

```diff
@@ -11,13 +11,20 @@ jobs:
   update-from-upstream:
     runs-on: ubuntu-latest
     steps:
+      - name: Generate GitHub App token
+        uses: tibdex/github-app-token@v1
```

Comment:
Define which version of the tibbex/github-app-token you are using. 
`@v1` will always take the latest version of v1 tags.

Reply:
We do use `tibdex/github-app-token@v1` in other workflows as well. Do you want me to add code that makes the version more specific in all places to prevent possible foul play (e.g. by pushing a `1.666.0` that steals all of our stuff)? It doesn't seem to make sense to do it only here if that's what you're afraid of. :)

Reply:
I misunderstood your issue. I thought it was due to a breaking change in tibdex/github-app-token. 
Stay with v1